exchange/bidder_test.go

```diff
@@ -516,9 +517,19 @@ func TestMultiCurrencies(t *testing.T) {
 		currencyConverter := currencies.NewRateConverter(
 			&http.Client{},
 			mockedHTTPServer.URL,
-			time.Duration(10)*time.Second,
+			// time.Duration(10)*time.Second,
+			time.Duration(10)*time.Millisecond,
 			time.Duration(24)*time.Hour,
+			clock.NewRealClock(),
 		)
+		time.Sleep(time.Duration(500) * time.Millisecond)
+		currencyConverterTickerTask := currencies.NewTickerTask(time.Duration(10)*time.Second, currencyConverter)
```

Comment:
This test is verifying the `requestBid` function when provided with valid rate conversions. We don't need to run the timer here. We can set up our conversions by creating a new rate converter, call the converter method `Run()` once so that the mocked rates are stored in the converter state and then calling `Rates()` on the converter to retrieve the conversions to be passed into `requestBid`.